components/Login/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Form, Input, Button, Checkbox, Typography } from "antd";
+import { MailOutlined, LockOutlined } from "@ant-design/icons";
+import { useAuth } from "~/components/Auth";
+import Link from "next/link";
+
+import styles from "./style.module.css";
+
+function LoginForm() {
```

Comment:
```suggestion
function Login() {
```

This component should be just `Login` to match the component folder, as a good practice.